experimental/cloudstrapper/README.md

```diff
@@ -0,0 +1,178 @@
+
+# Chapter 1
+
+## 0. Prerequisites
+   - Create private repo on github to host helm charts. 
+     This information would be used in the build.yaml and cluster.yaml files when building and deploying Magma respectively. 
+
+   - Identify two security keys in the Build/Gateway regions to be used for the following
+     - Bootkey: Used only by Cloudstrapper instance
+     - Hostkey: Used by all Gateway instances
+       Both values are specified in the defaults.yaml vars file and embedded in hosts.
+     To generate keys through a playbook, see section 1 below.
+
+   - Create inventory directory on localhost to save keys, secrets etc. This directory
+     will be referred to as WORK_DIR and used as dirInventory in commands.
+
+   - Gather following credentials and update secrets.yaml on local machine in WORK_DIR. 
+     Use format from $CODE_DIR/playbooks/roles/vars/secrets.yaml as base.
+     - AWS Access and Secret keys
+     - Github username and PAT
+     - Dockerhub username and password 
+
+   - Understand key directories
+     - CODE_DIR: Directory hosting Magma code, typically in the ~/code/magma/experimental/cloudstrapper folder
+     - VARS_DIR: Directory where all variables reside, typically in the CODE_DIR/playbooks/roles/vars folder
+     - WORK_DIR: Directory where all working copies reside, typically n the ~/magma-experimental
```

Comment:
The reason I called it is experimental to indicate this is a new feature for users to test and based on traction, it can find it's way into the main source directories. In case the idea doesn't resonate with the audience, this can be removed from the experimental directory without any impact to the primary source directories.